keyboards/skme/zeno/info.json

```diff
@@ -1,18 +1,19 @@
+           "layout": [{"label":"0,2", "x":2.6, "y":0.235}, {"label":"0,0", "x":0.6, "y":0.25}, {"label":"0,1", "x":1.6, "y":0.25}, {"label":"0,11", "x":12.75, "y":0.28}, {"label":"0,12", "x":13.75, "y":0.30000000000000004}, {"label":"0,13", "x":14.75, "y":0.30000000000000004}, {"label":"1,14", "x":15.75, "y":0.30000000000000004}, {"label":"1,0", "x":0.5, "y":1.25, "w":1.5}, {"label":"1,1", "x":2, "y":1.25}, {"label":"1,10", "x":12.4, "y":1.28}, {"label":"1,11", "x":13.4, "y":1.3}, {"label":"1,12", "x":14.4, "y":1.3}, {"label":"1,13", "x":15.4, "y":1.3, "w":1.5}, {"label":"2,0", "x":0.4, "y":2.25, "w":1.75}, {"label":"2,1", "x":2.15, "y":2.25}, {"label":"2,10", "x":12.8, "y":2.3000000000000003}, {"label":"2,11", "x":13.8, "y":2.3000000000000003}, {"label":"2,12", "x":14.8, "y":2.3000000000000003, "w":2.25}, {"label":"3,0", "x":0.25, "y":3.25, "w":2.25}, {"label":"3,1", "x":2.5, "y":3.25}, {"label":"3,10", "x":12.4, "y":3.3000000000000003}, {"label":"3,11", "x":13.4, "y":3.3000000000000003}, {"label":"3,12", "x":14.4, "y":3.3000000000000003, "w":1.75}, {"label":"3,14", "x":16.15, "y":3.3000000000000003}, {"label":"4,0", "x":0.25, "y":4.3, "w":1.25}, {"label":"4,1", "x":1.5, "y":4.3, "w":1.25}, {"label":"4,12", "x":14.65, "y":4.3, "w":1.25}, {"label":"4,14", "x":15.9, "y":4.3, "w":1.25}, {"label":"0,3", "x":0, "y":1.3000000000000003}, {"label":"0,4", "x":1, "y":1.3000000000000003}, {"label":"0,5", "x":2, "y":1.3000000000000003}, {"label":"0,6", "x":3, "y":1.3000000000000003}, {"label":"1,2", "x":-0.5, "y":2.3000000000000003}, {"label":"1,3", "x":0.5, "y":2.3000000000000003}, {"label":"1,4", "x":1.5, "y":2.3000000000000003}, {"label":"1,5", "x":2.5, "y":2.3000000000000003}, {"label":"2,2", "x":-0.25, "y":3.3000000000000003}, {"label":"2,3", "x":0.75, "y":3.3000000000000003}, {"label":"2,4", "x":1.75, "y":3.3000000000000003}, {"label":"2,5", "x":2.75, "y":3.3000000000000003}, {"label":"3,2", "x":0.25, "y":4.3}, {"label":"3,3", "x":1.25, "y":4.3}, {"label":"3,4", "x":2.25, "y":4.3}, {"label":"3,5", "x":3.25, "y":4.3}, {"label":"4,3", "x":0.5, "y":5.3}, {"label":"4,5", "x":-7.8, "y":8.45, "w":2.75}, {"label":"0,7", "x":-3.75, "y":3.3}, {"label":"0,8", "x":-2.75, "y":3.3}, {"label":"0,9", "x":-1.75, "y":3.3}, {"label":"0,10", "x":-0.75, "y":3.3}, {"label":"1,6", "x":-4.25, "y":4.3}, {"label":"1,7", "x":-3.25, "y":4.3}, {"label":"1,8", "x":-2.25, "y":4.3}, {"label":"1,9", "x":-1.25, "y":4.3}, {"label":"2,6", "x":-3.95, "y":5.3}, {"label":"2,7", "x":-2.95, "y":5.3}, {"label":"2,8", "x":-1.9500000000000002, "y":5.3}, {"label":"2,9", "x":-0.9500000000000002, "y":5.3}, {"label":"3,6", "x":-4.45, "y":6.3}, {"label":"3,7", "x":-3.45, "y":6.3}, {"label":"3,8", "x":-2.45, "y":6.3}, {"label":"3,9", "x":-1.4500000000000002, "y":6.3}, {"label":"4,7", "x":-4.45, "y":7.3, "w":2}, {"label":"4,8", "x":-2.45, "y":7.3}, {"label":"4,9", "x":-1.4500000000000002, "y":7.3, "w":1.25}]
```

Comment:
A precision of `0.30000000000000004` here is rather pointless....

Reply:
Precision updated to 4 trailing digits in following commit